File: Java Food Order App/GuestCustomer.java
package FoodOrderApp;

public class GuestCustomer extends Customer implements Bill {
	static String type = "Guest Customer";
	
	GuestCustomer(byte customerID, Name customerName, long customerPhone, Address customerAddress, String customerEmail) {
		super(customerID, customerName, customerPhone, customerAddress, customerEmail);
	}
	public void payBill(float total) {
		System.out.println("\tTotal Price is: " + total);
		System.out.println("\n\tSince you are our " + GuestCustomer.type);
		System.out.println("\tAfter Discount: " + total);
	}
}

File: Kotlin Food Order App/GuestCustomer.kt
package foodOrderApp


class GuestCustomer(
    customerID: Byte,
    customerName: Name,
    customerPhone: Long,
    customerAddress: Address,
    customerEmail: String
) : Customer(customerID.toByte(), customerName, customerPhone, customerAddress, customerEmail), BillOrder
{

    override fun payBill(total: Float) {
        println("\tTotal Price is: $total")
        println("\n\tSince you are our $type")
        println("\tAfter Discount: $total")
    }

    companion object {
        var type = "Guest Customer"
    }
}